packages/vaex-core/setup.py

```diff
@@ -183,12 +183,14 @@ def __str__(self):
       include_package_data=True,
       ext_modules=([extension_vaexfast] if on_rtd else [extension_vaexfast, extension_strings, extension_superutils, extension_superagg]) if not use_skbuild else [],
       zip_safe=False,
+      python_requires=">=3.6",
```

Comment:
@maartenbreddels to disable wheel builds for cp36 and cp37:
```suggestion
      python_requires=">=3.8",
```